src/ImageSharp/Formats/Jpeg/Components/Decoder/JpegFrame.cs

```diff
@@ -95,14 +95,23 @@ public void Dispose()
         /// </summary>
         public void InitComponents()
         {
-            this.McusPerLine = (int)MathF.Ceiling(this.SamplesPerLine / 8F / this.MaxHorizontalFactor);
-            this.McusPerColumn = (int)MathF.Ceiling(this.Scanlines / 8F / this.MaxVerticalFactor);
+            this.McusPerLine = (int)MathF.Ceiling(this.PixelWidth / 8F / this.MaxHorizontalFactor);
```

Comment:
The values are all (positive) `int`s, so this can be written to avoid `(int)MathF.Ceiling` as
```c#
uint tmp = (uint)this.MaxHorizontalFactor * 8;
this.McusPerLine = (int)(((uint)this.PixelWidth + tmp - 1) / tmp);
```
[Sharplab](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKBuIGYACMxgYUZoG9GePrfmTYikYBJAHYBLDOQAUASj4DONAQIwALSbgB0AWTABXXAAUYUADKTxMRgF5Gs6xnl7smgGI7WMSQBtrAHNZTW0dE0kEGD8AdUkAE01GAHpGAA4PFMZQ3TcEAAloSQAvCHEMbD8PbDAMaHkAblVeAF9mnnbBZhEJaVIFTpV+NR5nbPwAB3tsrVzsAqLS8srq2uhGACp0puGRnP0jU3MrG2mQ2fDI6LjEjUYAanGpuEZyRVSMSZ2BNt2lXgY3TEUgw9AGfyGIx4hjGnymDlkMPK8n2eUKUBKZQqVRqdSgm22nXUFwMxjMlmstgRznksjpSJc+wiUViCSSjzhjBebyycManV+Ak6gLGzOubLu3ECMAwDUYuBlcsFAKYYzRiyxK1x6ylivleuVPBF5UYpKOFNOutl+utvxaQA==)

Reply:
@JimBobSquarePants can we get this as some `int DivideCeil(int number, int divisor)` method to `Numerics.cs`? Placing it somewhere in the `JpegDecoderCore` or even in a separate `JpegUtils` file seems odd.

Reply:
Sounds good to me!